success-criteria.md

```diff
@@ -113,34 +113,19 @@ behind this decision ought to be thoroughly documented.
 view](https://www.w3.org/Consortium/Points/)[POINTS]: that of individual web
-users (both in aggregate and individually), publishers and their partners (both
+users (both in aggregate and individually), --publishers and their partners (both
 authors and the business model that funds them) and the delivery access
-mechanism (both connectivity and navigation).
-
-Any technology can be abused. Open societies do not attempt to suppress
-technology, but rather put [appropriate
-regulations](https://iabeurope.eu/all-news/iab-europes-press-statement-openrtb-and-eu-data-protection-law)
-[OPEN-RTB-EU-LAW] in place to define acceptable and unacceptable uses of that
-technology. For example, automobiles are not required to integrate functionality
-that technically prevents them from exceeding the speed limit. Instead, drivers
-are educated and trained in traffic rules, and drivers who violate speed limits
-are subject to fines and/or deprived of their permits. However, documenting
-specific criteria as to what constitutes a violation helps enable easier
-detection and reporting of non-compliance with the regulations that govern
-technology.
+mechanism (both connectivity and navigation).--
+
+Any technology can be abused. Any technology can be abused. Open societies consider all technologies but define ethical and unethical uses and regulate or restrict accordingly. Documenting specific criteria as to what constitutes an ethical use of a technology helps enable easier detection and reporting of non-compliance with the regulations that govern societies and support the process of constructing such regulations.
 
 By documenting the defined norms and principles behind appropriate and
 inappropriate data collection and processing, we can better devise methods of
 accountability. This accountability requires each participant that has access to
 data collection and processing to abide by its responsibility not to abuse the
 data under its control. This in turn requires definitions of legitimate data
 collection and processing as well as transparency around whether the data
-controller has fulfilled its obligations. One of the first assumptions we
-document is that an advertising-funded business model supports the open web, and
-hence any changes that degrade the efficacy of this business model negatively
-impacts end users. While end users increasingly understand advertising funds
-their free access to the open web, they desire improved transparency and control
-over their personal data.
+controller has fulfilled its obligations. One of the first assumptions we document is that an advertising-funded business model supports the open web and is a business model that positively impacts end users. End users increasingly understand advertising funds their free access to the open web. However, they join politicians, publishers and advocates in a desire for improved transparency and individual control over end user personal data.
```

Comment:
'degrade the efficacy' is poorly defined and implies that the current suite of proposals would do so, but that is not clearly proven. Propose we rephrase.

Reply:
The section starting with 'While end users...' does not encompass all relevant interests. We should add detail here to be clear about all the interests we wish to understand and handle with any feedback within this group.

Reply:
> JMK: Multiple studies have shown that as you degrade the accuracy, granularity, and timeliness of the data feeding algorithms the worse they perform -- hence the efficacy of marketers' algorithms will be degraded as we reduce the quality of inputs into them.  

> I agree with the the rephrasing, but suggest it is incomplete as it leaves out the above point.

This comes back to the top point - the worse they perform compared to what? I don't think a 'worse' or 'better' performance is the primary concern we need to engage with so much as the processes that exist and if we need them to continue. I am not sure it is useful to try and set the bar at 'zero degradation'. As such, unless you have a more specific phrasing here to consider, I think we should leave this edit as is.